include/psen_scan_v2/raw_processing.h

```diff
@@ -25,8 +25,15 @@
 
 namespace psen_scan_v2
 {
+/**
+ * @brief Namespace containing functions, etc. needed to transform
+ * higher level data to raw data (sent to or received from scanner) or vice versa.
```

Comment:
Data data data data?
```suggestion
 * higher level interpretation to raw data (sent to or received from scanner) or vice versa.
```